docs/pages/reference/mesheryctl-commands.md

```diff
@@ -369,6 +369,20 @@ Installation, troubleshooting and debugging of Meshery and its adapters.
       </tr>
       {% endfor %}
     {% endfor %}
+    {% for subcommand_hash in site.data.mesheryctlcommands.pattern.delete.commands %}{% assign subcommand = subcommand_hash[1] %}
+      <tr>
+        <td><a href="{{ site.baseurl }}/reference/mesheryctl/pattern/delete">{{ subcommand.name }}</a></td>
+        <td></td>
+        <td>{{ subcommand.description }}</td>
```

Comment:
Outside of the need to reference "pattern.delete", why are there changes in these lines? These should all be taken from template.

Reply:
@leecalcote Sir, I think this exist in every steps!